src/components/Stories/StoryInfo.tsx

```diff
@@ -0,0 +1,189 @@
+import {
+    Box,
+    Button,
+    DropdownMenu,
+    Text,
+    colors
+} from '@impact-market/ui';
+import { useRouter } from 'next/router';
+import CanBeRendered from '../CanBeRendered';
+import React from 'react';
+import String from '../../libs/Prismic/components/String';
+import Trim from '../Trim';
+import styled from 'styled-components'
+import useFilters from '../../hooks/useFilters';
+
+const mq = {
+    bigTablet: `(max-width: 1200px)`,
+    desktop: `(max-width: 1500px)`,
+    phone: `(max-width: 500px)`,
+    smallTablet: `(max-width: 750px)`
```

Comment:
We should use as breakpoints, the same values that we use on the UI, otherwise we'll have layouts breaking at different breakpoints than the rest of the app, this can be troublesome